pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -733,6 +735,11 @@ func generateServerURL(serverIP string, nodePort int32) (string, error) {
 }
 
 // SupportedStorageMode Return install etcd supported storage mode
-func SupportedStorageMode() []string {
-	return []string{etcdStorageModeEmptyDir, etcdStorageModeHostPath, etcdStorageModePVC}
+func SupportedStorageMode() sets.String {
```

Comment:
set.String is deprecated, how about use `sets.Set[string]` to replace it?

Reply:
> how about use sets.Set[string] to replace it?

ok